packages/fether-react/src/Health/Health.js

```diff
@@ -43,15 +48,19 @@ class Health extends Component {
 
   statusToFriendlyMessage = () => {
     const {
-      health: { status, payload }
+      health: { status, payload },
+      chainName
     } = this.props;
+
+    const chainNameAppend = isLoading(chainName) ? '' : ` (${chainName})`;
```

Comment:
or `pipe(withoutLoading)`? Less react renders.

Reply:
Following Thibaut's first comment I thought that it was maybe due to the chainName value not having been updated (because we switched network and the chainName is now loading), so I felt it was safer to not display anything if it's loading rather than the old chain name. But I don't think this was necessarily the cause; I don't mind using `pipe(withoutLoading)`

Reply:
I don't know if we receive the chainName immediately or not (I don't know what's the logic for it or what we can assume); if the chainName$ can take a while to respond, then the component wouldn't render until we receive it, which would be annoying

Also as a side note, we should probably make chainName$ use `parity_chain` rather than `parity_netChain` : https://github.com/paritytech/wiki/blob/master/JSONRPC-parity-module.md#parity_netchain
PR here https://github.com/paritytech/js-libs/pull/33